### PR TITLE
Change type of params in rent card - review 1

### PR DESCRIPTION
**Pull request by Usiel-Gandalf**
Change type of params the LPPaymentCard

With this i change the types of params the LPPaymentCard, it change to String kind of param
Important Note: i have to modify the ApartmentView only in the part in what i call the LPPaymentCard, where i was sending the data, because when i change the type of param shown error for the change, but just change that.

Change type of params

https://devaptivist.atlassian.net/browse/LP-178?atlOrigin=eyJpIjoiYWZjZTIyMzRhZTJlNGM2NWI0Y2I4YTE4MTkwOTY1MjQiLCJwIjoiaiJ9

### CODE REVIEW

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 1]
```diff
@@ -422,9 +422,9 @@ fun LPGenericElevatedCardImage(
 @Composable
 fun LpPaymentsRentCard(
     modifier: Modifier,
-    month: String,
-    quantity: Double,
-    date: Timestamp,
+    currentMonth: String,
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:432]: Please update the documentation

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 2]
```diff
@@ -469,12 +469,12 @@ fun LpPaymentsRentCard(
             )
             Column(modifier = Modifier.padding(start = LeasePertTheme.sizes.midLargeSize)) {
                 Text(
-                    text = "$$quantity",
+                    text = "$$quantityToPay",
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: We can change the string and get it from the resources and format with the `%.2f` example:
<string name="welcome_messages" >
        Hello, %s! You have %d new messages. Also %.2f
    </string>

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 1 above)
Comment (Usiel-Gandalf) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:432]: Done

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 2 above)
Comment (Usiel-Gandalf) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: I don't understand you well what are you try to saying, did you mean what the symbols must be a string resource?, i do that and i hpe your commentaries @melissaquezada94

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 3]
```diff
@@ -469,12 +469,12 @@ fun LpPaymentsRentCard(
             )
             Column(modifier = Modifier.padding(start = LeasePertTheme.sizes.midLargeSize)) {
                 Text(
-                    text = "$$quantity",
+                    text = stringResource(id = R.string.plus_symbol_for_quantities) + "$quantityToPay",
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Please check the "formatted strings resources"

Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: we need to concatenate all the value in the string from the strings file please

File: core/designsystem/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -16,4 +16,5 @@
     <string name="text_choose_file">Choose File</string>
     <string name="text_no_selected_file">No selected file…</string>
     <string name="text_unable_to_get_file_name">Unable to get file name</string>
+    <string name="plus_symbol_for_quantities">+</string>
```
Comment (carlosph816) [inline core/designsystem/src/main/res/values/strings.xml:19]: we need to add the new values that is going to be concatenated here

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 3 above)
Comment (Usiel-Gandalf) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Done

File: core/designsystem/src/main/res/values/strings.xml (hunk 4 above)
Comment (Usiel-Gandalf) [reply core/designsystem/src/main/res/values/strings.xml:19]: Done

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt (hunk 3 above)
Comment (Usiel-Gandalf) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: Done

**Review by irv205 — APPROVED** [review]

**Review by carlosph816 — APPROVED** [review]

**Pull request merged**